packages/iov-cosmos/src/cosmoscodec.ts

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable @typescript-eslint/camelcase */
+import {
+  Address,
+  ChainId,
+  Identity,
+  Nonce,
+  PostableBytes,
+  PrehashType,
+  SignableBytes,
+  SignedTransaction,
+  SigningJob,
+  TransactionId,
+  TxCodec,
+  UnsignedTransaction,
+} from "@iov/bcp";
+import { Sha256 } from "@iov/crypto";
+import { Encoding } from "@iov/encoding";
+import { marshalTx, unmarshalTx } from "@tendermint/amino-js";
+
+import { isValidAddress, pubkeyToAddress } from "./address";
+import { parseTx } from "./decode";
+import { buildSignedTx, buildUnsignedTx } from "./encode";
+
+const { toHex, toUtf8 } = Encoding;
+
+function sortJson(json: any): any {
+  if (typeof json !== "object" || json === null) {
+    return json;
+  }
+  if (Array.isArray(json)) {
+    return (json as readonly any[]).map(sortJson);
+  }
+  const sortedKeys = Object.keys(json).sort();
+  const result = sortedKeys.reduce(
+    (acc, k) => ({
```

Comment:
Can we afford a few more characters for `accumulator?` and `key?`? I think it saves a good amount of brain power when reading the code